dynawo/cmake/SetPatchCommands.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+function( SetPatchCommands projectName)
```

Comment:
Please follow the coding style in the file (top CMakeLists.txt, even with the same keyword you are not consistent through out the file

Reply:
And if you define it here you could already replace the same code in the top CMakeLists with the call of this function

Reply:
I do not see what you mean exactly. I want to follow the coding style. About that function, it is a new one in replacement of the one inside the very big CMakeLists.txt. It took me very much time to understand it. What I wish to do is to avoid making that file bigger.

Reply:
I know it is a replacement but I don't see the code deleted in the top CMakeLists.txt

For the coding style, we don't use space after (, so function(foo),
also if(Test) and not if (Test) or if ( Test) as you use both

Reply:
I will change, no problem.

Reply:
powsybl does not have our coding style

Reply:
I know. the two projects are independant